### PR TITLE
Rename CINECA to GECKO and add metadata

### PR DESCRIPTION
**Pull request by beckyjackson**
See #3

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,24 +54,26 @@ build/robot-validate.jar: | build
 build/robot-rdfxml.jar: | build
 	curl -Lk -o $@ https://build.obolibrary.io/job/ontodev/job/robot/job/mireot-rdfxml/lastSuccessfulBuild/artifact/bin/robot.jar
 
-### CINECA Tasks
+### GECKO Tasks
 
-data/cineca.tsv:
```
Comment (jamesaoverton) [inline Makefile]: It's a minor thing, but I think that `data/cineca.tsv` should not be renamed to "gecko". @mcourtot?

File: metadata/gecko.ttl [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/>.
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<>
+  rdf:type owl:Ontology ;
+  dcterms:title "Genomics Cohorts Knowledge Ontology" ;
+  dcterms:description "An ontology to represent genomics cohort attributes. The GECKO is maintained by the CINECA project, https://www.cineca-project.eu, and standardises attributes commonly used for genomics cohort description as well as individual-level data items. A series of tools is being developed to enable automated generation of harmonised data files based on a JSON schema mapping file. For more information please contact info@cineca-project.eu" ;
+  dcterms:license <https://creativecommons.org/licenses/by/4.0/> ;
+  dcterms:rights "You are free to:
```
Comment (jamesaoverton) [inline metadata/gecko.ttl]: I think that @mcourtot would prefer to just use the license, and include rights.

File: Makefile (hunk 1 above)
Comment (mcourtot) [reply Makefile]: sounds right - it helps showing the provenance from cineca in that case. agree it is minor too.

File: metadata/gecko.ttl (hunk 2 above)
Comment (mcourtot) [reply metadata/gecko.ttl]: I'm not quite sure what to put into rights here - if you have text you can reuse please do. I'm still not sure why we need both as per https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1019#issuecomment-595836706, but not a gigantic issue.

Comment (jamesaoverton) [reply metadata/gecko.ttl]: Ok, then I prefer to just use license and remove rights, for now.

Comment (mcourtot) [reply metadata/gecko.ttl]: seems the OBO dashboard will warn if rights is not in, so suggest keep?

File: Makefile (hunk 1 above)
Comment (jamesaoverton) [reply Makefile]: True. Let’s keep it.

Comment (jamesaoverton) [reply Makefile]: Sorry, I mixed up my replies! Yes let’s call this table cineca.

Comment (beckyjackson) [reply Makefile]: Fixed

File: metadata/gecko.ttl (hunk 2 above)
Comment (jamesaoverton) [reply metadata/gecko.ttl]: No, the dashboard only requires license. Your call!

**Pull request merged**